config/default.ts

```diff
@@ -17,6 +17,8 @@ export default {
     priceFeed: process.env.priceURL || "http://localhost:8090/v1/getPrice",
     tokenInfoFeed:
       process.env.tokenInfoURL || "http://localhost:8091/v1/getTokenInfo",
+    fingerprintInfoFeed:
+      process.env.fingerprintInfoUrl || "http://localhost:8091/v1/getFingerprintInfo",
```

Comment:
This envvar needs to be added to the production deployment I would assume? @szandala

Reply:
In env OR takes default value "http://localhost:8091/v1/getFingerprintInfo"

Reply:
so do we need to a set an env var in production? Or is it okay to just keep the default? The default works on my local machine since I have the service running on localhost